Dockerfile.foreman

```diff
@@ -0,0 +1,18 @@
+FROM python:3.5-slim
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
```

Comment:
I've observed that a number of good practices from the world before containerization no longer apply or are turned on their heads by something like Docker. This is one: I think a generic account name is good here. The point is to simplify the infrastructure to the point of being as generic as possible.